lib/orderbook/types.ts

```diff
@@ -1,6 +1,11 @@
 import { SwapClientType } from '../constants/enums';
 import { SwapSuccess, SwapFailure } from '../swaps/types';
 
+export type Limits = {
```

Comment:
I don't think we need a new type for this. We can just create an object with hard-coded properties and the compiler will know that they're there.